src/core/execute.h

```diff
@@ -226,7 +226,6 @@ static inline bool exec_context_restrict_namespaces_set(const ExecContext *c) {
 }
 
 typedef enum ExecFlags {
-        EXEC_CONFIRM_SPAWN     = 1U << 0,
         EXEC_APPLY_PERMISSIONS = 1U << 1,
         EXEC_APPLY_CHROOT      = 1U << 2,
         EXEC_APPLY_TTY_STDIN   = 1U << 3,
```

Comment:
btw, if you drop usage of one bit here, please move the others one up, so that all bits are used here... Not that it really matters, but me OCD prefers if we use all bits possible. Given that this is not a public API making such a change is unproblematic.